BootCampV2/Homeworks/ThirdWeek/Command/LoginCommand.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+
+
+namespace BootCampV2.Homeworks.ThirdWeek.Command
+{
+    public class LoginCommand : ICommand
+    {
+        private readonly UserFileManager _userFileManager;
+
+        private string _prefix = "-l";
+
+        public string Prefix
+        {
+            get
+            {
+                return _prefix;
+            }
+
+            set
+            {
+                _prefix = value;
+            }
+        }
+
+        private string _description = "-l (Username) (Password)";
+
+        public string Description
+        {
+            get
+            {
+                return _description;
+            }
+            set
+            {
+                _description = value;
+            }
+        }
```

Comment:
Use auto properties. Prefer to use get only property, this should not change over time.